libraries/chain/market_evaluator.cpp

```diff
@@ -189,6 +175,7 @@ void_result call_order_update_evaluator::do_apply(const call_order_update_operat
       {
          d.modify(_paying_account->statistics(d), [&](account_statistics_object& stats) {
                stats.total_core_in_orders += o.delta_collateral.amount;
+               assert(dynamic_asset.current_supply <= 0);
```

Comment:
This is apparently wrong.
By the way don't use `assert()`, see https://github.com/bitshares/bitshares-core/issues/511